AISKU/src/Initialization.ts

```diff
@@ -12,6 +12,29 @@ import { AjaxPlugin as DependenciesPlugin, IDependenciesPlugin } from '@microsof
 
 "use strict";
 
+/**
+ * Trys to add an event handler for the specified event to the window, body and document
+ * @param eventName {string} - The name of the event
+ * @param callback {any} - The callback function that needs to be executed for the given event
+ * @return {boolean} - true if the handler was successfully added
+ */
+function addEventHandler(eventName: string, callback: any): boolean {
```

Comment:
Move this to Core Utils, if we really want to attach to both window and window["body"] everywhere

Reply:
For the normal window events then yes we would, but this is not the case for all events...

Reply:
We do have multiple scenarios for window events, having this in Core instead of SKU code would better so it can be reused

Reply:
Moved to CoreUtils